### DISCUSSION TITLE
Floating table of contents in distill posts?

### DISCUSSION THREAD

**Discussion by eigentaylor** [alshedivat/al-folio#504] [Q&A]
I believe distill has a `toc_float` option, but adding `toc_float: true` into the markdown file of my post does not seem to do anything. Having the table of contents follow as you scroll through lengthy posts would be _very_ convenient.

I suppose an alternate solution would be a "scroll to top" floating button, but I'm afraid I'm not experienced enough to add that to the footer or layout template without breaking everything (I did try and it didn't go well).

**Comment by saguileran** [comment]
Do you find any solution or is it unsolved yet?

**Reply by george-gca** [reply]
Adding changes to the distill template is something quite difficult, mainly because of how the [distill template works](https://github.com/alshedivat/al-folio/tree/main/assets/js/distillpub). Also  I don't think this is supported on the official template.
